src/isolate_setup.cpp

```diff
@@ -17,6 +17,19 @@ extern const uint8_t kDartCoreIsolateSnapshotData[];
 extern const uint8_t kDartCoreIsolateSnapshotInstructions[];
 }
 
+namespace {
+    class DllIsolateGroupData : public IsolateGroupData {
+        public:
+            DllIsolateGroupData(const char* url,
+                const char* packages_file,
+                AppSnapshot* app_snapshot,
+                bool isolate_run_app_snapshot)
```

Comment:
I would initialize `callback_data` as part of the constructor here instead of having it set as a member, then you shouldn't need the initializer to an empty struct below.